lib/less-browser/index.js

```diff
@@ -187,20 +187,28 @@ module.exports = function(window, options) {
     less.unwatch = function () {clearInterval(less.watchTimer); this.watchMode = false; return false; };
 
     //
-    // Get all <link> tags with the 'rel' attribute set to "stylesheet/less"
+    // Synchronously get all <link> tags with the 'rel' attribute set to
+    // "stylesheet/less".
     //
-    less.registerStylesheets = function() {
-        return new Promise(function(resolve, reject) {
-            var links = document.getElementsByTagName('link');
-            less.sheets = [];
-
-            for (var i = 0; i < links.length; i++) {
-                if (links[i].rel === 'stylesheet/less' || (links[i].rel.match(/stylesheet/) &&
-                    (links[i].type.match(typePattern)))) {
-                    less.sheets.push(links[i]);
-                }
+    less.registerStylesheetsImmediately = function() {
```

Comment:
you don't call `less.registerStylesheetsImmediately` from anywhere? and you have removed the call to `less.registerStylesheets` and that isn't called anywhere..

I guess you just need to call `less.registerStylesheetsImmediately` from bootstrap ?

otherwise, many thanks for tracking it down and fixing.